packages/front-end/components/Settings/EditDataSourceSettingsForm.tsx

```diff
@@ -5,10 +5,12 @@ import track from "../../services/track";
 import Modal from "../Modal";
 import { DataSourceInterfaceWithParams } from "back-end/types/datasource";
 import Field from "../Forms/Field";
+import Code from "../Code";
```

Comment:
Restoring this file is fine for now.  I don't think we actually need to render this anymore, but we can delete and clean up old references in a follow up PR.

Reply:
I've added a ticket to clean this up: https://github.com/growthbook/growthbook/issues/584 - we can add it to tech debt tasks to do later.

Reply:
Marking as resolved as it's been deleted in a recent commit since it isn't being used.